ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--- Provide a general summary of the issue in the Title above -->
```

Comment:
Should we add some comment about what to do if one wants to request an enhancement rather than report a bug? ("if you want to request an enhancement / a new feature, then feel free to disregard the following template", maybe followed by something like "please explain why it would be a good idea, and concentrate on the needs this would solve rather than how it would solve them and/or sketching an API")